staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -151,6 +152,10 @@ func (s *EtcdOptions) AddFlags(fs *pflag.FlagSet) {
 	fs.StringSliceVar(&s.StorageConfig.ServerList, "etcd-servers", s.StorageConfig.ServerList,
 		"List of etcd servers to connect with (scheme://ip:port), comma separated.")
 
+	fs.DurationVar(&s.StorageConfig.AutoSyncInterval, "etcd-auto-sync-interval", s.StorageConfig.AutoSyncInterval, ""+
```

Comment:
omit `default value is 0.`, the default is added automatically

Reply:
ok